src/MusicFestival.Vue.Template/Web.config

```diff
@@ -25,6 +25,9 @@
     <add key="ValidationSettings:UnobtrusiveValidationMode" value="None" />
     <add key="aspnet:UseTaskFriendlySynchronizationContext" value="true" />
     <add key="ContentApiClientValidationEnabled" value="true" />
+    <!-- siteinitialization calls already config.MapHttpAttributeRoutes(); so don't call it from content delivery api packages  -->
+    <add key="episerver:contentdeliverysearch:maphttpattributeroutes" value="false" />
```

Comment:
probably would camelCase these values

Reply:
Probaly good idea for consistency. Just a copy paste from Episerver instructions on page: https://world.episerver.com/documentation/developer-guides/content-delivery-api/configuration/#DefaultHTTPConfig

Reply:
I would leave as it is for now. Those key names with all lower case letters is the key the initialization modules use (it is true we could change the capitalization and it would still work).

Maybe the big question then would be should episerver be lower case in it or not (DXC samples use episerver (all letters lower case), comparing to aspnet and webpages also all lower case but if we compare to some other projects they use pascal casing for the whole key value.

Naming convention should be first established. I would leave this as is for now as it has no effect to the application working or not.